python/vast/vast/vast.py

```diff
@@ -76,8 +76,9 @@ def _next_batch():
 class VAST:
     """An instance of a VAST node."""
 
-    def __init__(self):
-        pass
+    def __init__(self, endpoint=None):
+        if endpoint is not None:
+            self.cli = CLI(endpoint=endpoint)
```

Comment:
Didn't try it, but this seems to imply that you leave `self.cli` as undefined when `endpoint` is `None`. That would then fail when you call `self.cli.xxx()`, no? Ideally, this should somehow be covered in the tests.

Reply:
> @rdettai maybe you have an idea why the thehive test fails with this?

This might actually be the reason

Reply:
Good catch!

Reply:
Looks like that was not the problem.